content/about/mirrors.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "GRASS Website mirrors"
+date: 2020-07-28T16:13:30+02:00
+layout: "general"
+---
+
+### Official master site via HTTP/FTP
+
+
+* [Official master site: grass.osgeo.org](http://grass.osgeo.org/)
+
+* We recommend cascaded mirroring: 
+
+  * Tier-1 mirrors: These mirrors read daily from the master site and might also act as rsync servers. 
+  * Tier-2 mirrors: These mirrors read from the tier-1 mirrors (these sites may be 1-2 day(s) behind the master site)
```

Comment:
This is no longer needed (may be removed)

Reply:
Ok, I've removed it.